python-sdk/pachyderm_sdk/api/pfs/utils.py

```diff
@@ -0,0 +1,20 @@
+from shutil import which
+from subprocess import run
+
+
+def check_pachctl(*, ensure_mount: bool = False) -> None:
```

Comment:
This was used for the mount functionality which got removed. We technically do not need this anymore